rust/src/web/graphql/stakes/mod.rs

```diff
@@ -38,18 +35,19 @@ impl StakesQueryRoot {
             None => return Ok(None),
         };
 
-        let AggregatedEpochStakeDelegations { delegations, .. } =
-            staking_ledger.aggregate_delegations()?;
+        // Delegations will be present if the staking ledger is
+        let delegations = db.get_delegations_epoch("mainnet", epoch)?.unwrap();
```

Comment:
Add a TODO for supporting both networks?

Reply:
I'd rather not litter TODOs in the codebase. That's what the project board is for.